.github/renovate.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:base"],
+  "labels": ["dependencies"],
+  "packageRules": [
+    {
+      "automerge": true,
```

Comment:
> the PR then goes through standard build checks before being reviewed and accepted by a team member.

Mh, unless I am misunderstand something, the documentation says that `"automerge": true` means it will try merge its PR automatically if all tests pass, and only request a review if tests fail.

https://docs.renovatebot.com/key-concepts/automerge/

Did you mean to add that?

Reply:
Hmm; good catch.  The intent is to block on a human reviewing.

they said, I’ve seen projects bitrot by not accepting updates.  Some of my stuff (like where I grabbed this from) automerge the more trivial changes they do pass a full build :)

Let me fix that, but which do you prefer?

Reply:
> Let me fix that, but which do you prefer?

Oh, I think whatever you prefer is fine.
I was just curious about that `automerge: true`, and then I noticed that what it does did not match your PR description, so I mentioned it. :)